### PR TITLE
chore: update local env comment

### PR DESCRIPTION
**Pull request by swatDong**
Per comment https://github.com/OfficeDev/TeamsFx/pull/6249#discussion_r990603764
Update comment wording in `.env.teamsfx.local`.

### CODE REVIEW

**Review by jidddddd — DISMISSED** [review]

**Review by XiaofuHuang — DISMISSED** [review]

File: packages/fx-core/src/plugins/resource/frontend/env.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ async function _saveEnvFile(envPath: string, envs: RemoteEnvs, configs: RemoteEn
   await fs.ensureFile(envPath);
 
   const envString =
-    `# Following variables are generated by TeamsFx${os.EOL}` +
+    `# TeamsFx will overwrite the following variable values when running debug${os.EOL}` +
```
Comment (hund030) [inline packages/fx-core/src/plugins/resource/frontend/env.ts:101]: Maybe `TeamsFx will overwrite the following variable values when provisioning`?

Comment (swatDong) [reply packages/fx-core/src/plugins/resource/frontend/env.ts]: Revert this file since it's for provision.

**Comment by kuojianlu** [comment]
Is it necessary to update https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/src/component/debugHandler/localEnvProvider.ts#L70-L76?

**Comment by swatDong** [comment]
> Is it necessary to update https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/src/component/debugHandler/localEnvProvider.ts#L70-L76?

Good catch! Updated.

**Review by kuojianlu — APPROVED** [review]

**Review by XiaofuHuang — APPROVED** [review]

**Pull request merged**